src/components/ProfilePage/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect, useContext } from "react";
+import GamesCarousel from "../GamesCarousel";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Col, Layout } from "antd";
+import profilePageImg from "../../images/profilePage/profilePageImg.png";
+import db from "../../firebaseConfig";
+import { v4 as uuidv4 } from "uuid";
+import { AuthContext } from "../../auth/authContext";
+
+const { Sider, Content } = Layout;
+const ProfilePage = () => {
+  const { t } = useTranslation();
+  const { currentUser } = useContext(AuthContext);
+  const currentUserId = currentUser ? currentUser.uid : null;
+  const [user, setUser] = useState([]);
+  const ref = db.firestore().collection("users");
+
+  const getUsers = () => {
+    ref.where("uid", "==", currentUserId).onSnapshot((querySnapshot) => {
```

Comment:
shouldn't you just be querying by the document id? i'm assuming you're storing users where the key is user id. you shouldnt have to filter either, you should be able to get it just by getting it using the key (the one that shows up in the firebase ui)